File: tests/benchmarks/cases/mixed_service_loop/c/benchmark_case.c
#include "benchmark_case.h"
#include "benchmark_support.h"

static ZrBenchInt zr_bench_case_mixed_service_loop_run(int scale) {
    return zr_bench_run_mixed_service_loop(scale);
}

const ZrBenchCaseDescriptor zr_bench_case_descriptor_mixed_service_loop = {
        "mixed_service_loop",
        "BENCH_MIXED_SERVICE_LOOP_PASS",
        zr_bench_case_mixed_service_loop_run
};

File: tests/benchmarks/cases/mixed_service_loop/java/benchmark_case.java
final class MixedServiceLoopCase {
    static final String NAME = "mixed_service_loop";
    static final String PASS_BANNER = "BENCH_MIXED_SERVICE_LOOP_PASS";

    private MixedServiceLoopCase() {}

    static long run(int scale) {
        return BenchmarkSupport.mixedServiceLoop(scale);
    }
}
